src/test/java/e2e/pages/RegisterPage.java

```diff
@@ -0,0 +1,34 @@
+package e2e.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class RegisterPage extends BasePage {
+
+    public RegisterPage() throws Exception {
+        super();
+        path = "register";
```

Comment:
See - you're not setting the unique here because you don't really need it - you can wait for any element on a page to be visible. The unique one is redundant in all pages IMO